notify/notify.go

```diff
@@ -41,6 +41,9 @@ type Notifier struct {
 	privateKey string     // Public key for accessing MailJet API.
 }
 
+// Kind represents a kind of notification.
+type Kind string
```

Comment:
Should we add a bunch of recommended kinds i.e.
```
const (
    BroadcastService  Kind = "broadcast-service"
    BroadcastAPI        Kind = "broadcast-api"
    BroadcastNetwork Kind = "broadcast-network"
// etc....
)
```

Reply:
I think we should leave this to clients and keep the notify package agnostic.

Reply:
Fair enough, it's a bit harder to enforce a particular scheme then however, but maybe that's OK.